src/components/Datepicker.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <div v-if="show">
-    <p class="text-base mb-[10px]">
-      {{ label }}<span v-if="isRequired">*</span>
-    </p>
+    <p class="text-base mb-[10px]">{{ label }}<span v-if="isRequired">*</span></p>
```

Comment:
We are using same class at multiple places. Let's define it once and use it everywhere.